matchms/similarity/CosineGreedy.py

```diff
@@ -42,17 +42,24 @@ class CosineGreedy:
 
     .. testoutput::
 
-        Cosine score is 0.52 with 1 matched peaks
+        Cosine score is 0.83 with 1 matched peaks
 
     """
-    def __init__(self, tolerance: float = 0.1):
+    def __init__(self, tolerance: float = 0.1, mz_power: float = 0.0,
+                 intensity_power: float = 1.0):
         """
         Parameters
         ----------
-        tolerance
-            Peaks will be considered a match when <= tolerance apart.
+        tolerance:
+            Peaks will be considered a match when <= tolerance apart. Default is 0.1.
+        mz_power:
```

Comment:
x**0 always gives 1. 

Is it correct if `mz_power=0 ` then mz is ignored during score generation? 
If yes, is that clear to library users or should this be documented?

Reply:
Thanks Stefan. I added such an explanation to ``CosineGreedy`` and ``ModifiedCosine``.